docs/pages/access-controls/guides/passwordless.mdx

```diff
@@ -0,0 +1,293 @@
+usernameless authentication for Teleport.
+
+## Prerequisites
+
+* A Teleport cluster with WebAuthn configured.
+  See the [Second Factor: WebAuthn](./webauthn.mdx) guide.
+* A hardware device with support for WebAuthn and resident keys.
+  As an alternative, you can use a Mac with biometrics / Touch ID.
+* A web browser with WebAuthn support. To see if your browser supports
+  WebAuthn, check the [WebAuthn Compatibility](
+  https://developers.yubico.com/WebAuthn/WebAuthn_Browser_Support/) page.
+* A signed and notarized `tsh` for Touch ID.
+  [Download the macOS tsh installer](../../installation/#macos).
+
+A Teleport cluster capable of WebAuthn is automatically capable of passwordless.
+
+## Step 1/2. Register
+
+Register your passwordless device using `tsh`:
+
+```code
+$ tsh mfa add
+# Choose device type [TOTP, WEBAUTHN, TOUCHID]: WEBAUTHN
+# Enter device name: bio
+# Allow passwordless logins [YES, NO]: YES
+# Tap any *registered* security key
+# Tap your *new* security key
+# MFA device "bio" added.
+```
+
+You may pick either `WEBAUTHN` or `TOUCHID` as the device type. Make sure to
+answer `YES` to "Allow passwordless logins".
+
+If you are using a hardware device, a passwordless registration will occupy a
+resident key slot. Resident keys, also called discoverable credentials, are
+stored in persistent memory in the authenticator (i.e., the device that is used
+to authenticate). In contrast, MFA keys which are encrypted by the authenticator
+and stored in the Teleport Auth Server.
+
+Regardless of your device type, passwordless registrations may also be used for
+regular MFA.
+
+<Admonition type="tip" title="Important">
+If you plan on relying exclusively on passwordless, it's recommended to register
+more than one device. A portable hardware device is ideal, as it can be shared
+between `tsh`, the Teleport Web UI, and different computers.
+</Admonition>
+
+<Details
+  title="Registering Touch ID for tsh"
+  opened={true}
+>
+  Touch ID registrations are isolated by application. A Touch ID registration
+  for `tsh` is different from a registration made from Chrome or Safari. You may
+  register the same Touch ID device from multiple applications to get
+  passwordless access in all of them.
+</Details>
+
+## Step 2/2. Authenticate
+
+Authenticate using your passwordless credential:
+
+```code
+$ tsh login --proxy=example.com --auth=passwordless
+# Tap your security key
+# > Profile URL:        https://example.com
+#   Logged in as:       codingllama
+#   Cluster:            example.com
+#   Roles:              access, editor
+#   Logins:             codingllama
+#   Kubernetes:         enabled
+#   Valid until:        2021-10-04 23:32:29 -0700 PDT [valid for 12h0m0s]
+#   Extensions:         permit-agent-forwarding, permit-port-forwarding, permit-pty
+```
+
+A fully passwordless cluster defaults to passwordless logins, making
+`--auth=passwordless` unnecessary. See [enable passwordless by default](
```

Comment:
I would replace the link with "See the next section to learn how to configure passwordless logins for your cluster". Otherwise, it's a bit disorienting to click this link and navigate one section down.

Reply:
Done, only rephrased to "See the next section to learn how to enable passwordless by default", which I think is a bit more accurate.